asm/macros/scrcmd.inc

```diff
@@ -1081,66 +1088,68 @@
     .short 198
     .endm
 
-    .macro ScrCmd_0C7 arg0
+    .macro CheckPlayerOnBike destVarID
     .short 199
-    .short \arg0
+    .short \destVarID
     .endm
 
-    .macro ScrCmd_0C8 arg0
+    ; If rideBike == TRUE, sets the player on the bike.
+    ; If rideBike == FALSE, removes the player from the bike.
+    .macro SetPlayerBike rideBike
     .short 200
-    .byte \arg0
+    .byte \rideBike
     .endm
 
     .macro ScrCmd_0C9 arg0
     .short 201
     .byte \arg0
     .endm
 
-    .macro ScrCmd_0CA arg0
+    .macro GetPlayerState destVarID
     .short 202
-    .short \arg0
+    .short \destVarID
     .endm
 
-    .macro ScrCmd_0CB arg0
+    .macro SetPlayerState state
     .short 203
-    .short \arg0
+    .short \state
     .endm
 
-    .macro ScrCmd_0CC
+    .macro ChangePlayerState
     .short 204
     .endm
 
-    .macro ScrCmd_0CD arg0
+    .macro BufferPlayerName strbufIndex
```

Comment:
**suggestion:** `strbufIndex` -> `templateParam`

Reply:
Would something like `templateArg` make more sense? That would align more closely to how the rest of the string template code refers to those.

Reply:
`templateArg` makes sense to me, especially since I modeled the terminology after Python's f-strings: https://docs.python.org/3/library/stdtypes.html#str.format